src/releng_docs/deploying.rst

```diff
@@ -1,146 +1,244 @@
+           
+           You can follow the progress in release PR
 
-- A ``releng-cache`` S3 bucket is created and read/write/delete permission
-  granted for all above created policies. This S3 bucket is used to hold binary
-  files of builds.
+               <LINK-TO-NEXT-RELEASE-PR>
 
-- S3 buckets need to be created before deploying new service
 
-TODO: write how certificates are handled once Route51 is configured
 
+           [1] https://github.com/mozilla-releng/services
 
-Heroku
-------
 
-We currently use a (quite some time in beta) Heroku feature that allows to run
-docker images on Heroku infrastructure. To manually push docker images build in
-:ref:`previous praragraph <deploying-docker>` you need to push docker images to
-Heroku's custom docker registry.
+.. _`Example of release PR`: https://github.com/mozilla-releng/services/pull/237
+.. _`Open next release PR`: https://github.com/mozilla-releng/services/compare/production...master
+.. _`Release Enginnering`: https://wiki.mozilla.org/ReleaseEngineering
+.. _`Release Management`: https://wiki.mozilla.org/Release_Management
 
-.. code-block:: bash
 
-    % make build-tool-push
-	./result-tool-push/bin/push \
-		`realpath ./result-docker-releng_clobberer` \
-		https://registry.heroku.com \
-		-u <HEROKU_USERNAME> \
-		-p <HEROKU_PASSWORD> \
-		-N <HEROKU_APP>/web \
-		-T latest
+.. _release-managers:
+
+Release Managers
+----------------
+
+- `Rok Garbas`_
+
+
+.. _service-owners:
+
+Service Owners
+--------------
+
++---------------------------+---------------------------+
++ Service                   | Owner(s)                  |
++===========================+===========================+
++ :ref:`releng_archiver`    | - `Rok Garbas`_           |
++---------------------------+---------------------------+
++ :ref:`releng_clobberer`   | - `Rok Garbas`_           |
++---------------------------+---------------------------+
++ :ref:`releng_docs`        | - `Rok Garbas`_           |
++---------------------------+---------------------------+
++ :ref:`releng_frontend`    | - `Rok Garbas`_           |
++---------------------------+---------------------------+
++ :ref:`releng_mapper`      | - `Rok Garbas`_           |
++---------------------------+---------------------------+
++ :ref:`releng_slavehealth` | - `Rok Garbas`_           |
++---------------------------+---------------------------+
++ :ref:`releng_tooltool`    | - `Rok Garbas`_           |
++---------------------------+---------------------------+
++ :ref:`releng_treestatus`  | - `Rok Garbas`_           |
++---------------------------+---------------------------+
++ :ref:`shipit_bot_uplift`  | - `Bastien Abadie`_       |
++---------------------------+---------------------------+
++ :ref:`shipit_frontend`    | - `Rok Garbas`_           |
++                           | - `Bastien Abadie`_       |
++---------------------------+---------------------------+
++ :ref:`shipit_pipeline`    | - ???                     |
```

Comment:
Actually, @catlee was working on this last time we talked.

@lundjordan / @nthomas-mozilla are working on ``shipit_taskcluster`` which is not yet in ``mozilla-releng/services``

I will ask this at the shipit meeting today.